tests/transformer/test_operators.py

```diff
@@ -0,0 +1,356 @@
+    assert result.errors == ()
+    assert result.warnings == []
+    assert result.used_names == {}
+
+
+@pytest.mark.parametrize(*c_eval)
+def test_BitOr(c_eval):
+    result = c_eval('60 | 13')
+    assert result.code is not None
+    assert result.errors == ()
+    assert result.warnings == []
+    assert result.used_names == {}
+
+
+@pytest.mark.parametrize(*c_eval)
+def test_BitXor(c_eval):
+    result = c_eval('60 ^ 13')
+    assert result.code is not None
+    assert result.errors == ()
+    assert result.warnings == []
+    assert result.used_names == {}
+
+
+@pytest.mark.parametrize(*c_eval)
+def test_Invert(c_eval):
+    result = c_eval('~60')
+    assert result.code is not None
+    assert result.errors == ()
+    assert result.warnings == []
+    assert result.used_names == {}
+
+
+@pytest.mark.parametrize(*c_eval)
+def test_LShift(c_eval):
+    result = c_eval('60 << 2')
+    assert result.code is not None
+    assert result.errors == ()
+    assert result.warnings == []
+    assert result.used_names == {}
+
+
+@pytest.mark.parametrize(*c_eval)
+def test_RShift(c_eval):
+    result = c_eval('60 >> 2')
+    assert result.code is not None
+    assert result.errors == ()
+    assert result.warnings == []
+    assert result.used_names == {}
+
+
+# Logical Operators
+
+
+@pytest.mark.parametrize(*c_eval)
+def test_In(c_eval):
+    result = c_eval('1 in [1, 2, 3]')
+    assert result.code is not None
+    assert result.errors == ()
+    assert result.warnings == []
+    assert result.used_names == {}
+
+    eval_result = eval(result.code)
+    assert eval_result is True
+
+
+@pytest.mark.parametrize(*c_eval)
+def test_NotIn(c_eval):
+    result = c_eval('4 not in [1, 2, 3]')
+    assert result.code is not None
+    assert result.errors == ()
+    assert result.warnings == []
+    assert result.used_names == {}
+
+    eval_result = eval(result.code)
+    assert eval_result is True
+
+
+# Identity Operator
+
+
+@pytest.mark.parametrize(*c_eval)
+def test_Is(c_eval):
+    result = c_eval('True is True')
+    assert result.code is not None
+    assert result.errors == ()
+    assert result.warnings == []
+    # assert result.used_names == {}
+
+    eval_result = eval(result.code)
+    assert eval_result is True
+
+
+@pytest.mark.parametrize(*c_eval)
+def test_NotIs(c_eval):
+    result = c_eval('1 is not True')
+    assert result.code is not None
+    assert result.errors == ()
+    assert result.warnings == []
+    # assert result.used_names == {}
+
+    eval_result = eval(result.code)
+    assert eval_result is True
+
+
+# Unary Operators
+
+
+@pytest.mark.parametrize(*c_eval)
+def test_UAdd(c_eval):
+    result = c_eval('+1')
+    assert result.code is not None
+    assert result.errors == ()
+    assert result.warnings == []
+    # assert result.used_names == {}
+
+    eval_result = eval(result.code)
+    assert eval_result == 1
+
+
+@pytest.mark.parametrize(*c_eval)
+def test_USub(c_eval):
+    result = c_eval('-1')
```

Comment:
In Python 2.7 the AST of this expression is `Module(body=[Expr(value=Num(n=-1))])`.
Only Python 3 has the `USub` node here.
`-a` renders `USub` in both Python versions.